.rubocop_todo.yml

```diff
@@ -95,13 +95,13 @@ RSpec/InstanceVariable:
     - 'spec/slack/real_time/client_spec.rb'
     - 'spec/slack/slack_spec.rb'
 
-# Offense count: 75
+# Offense count: 77
 # Configuration parameters: .
 # SupportedStyles: have_received, receive
 RSpec/MessageSpies:
   EnforcedStyle: receive
 
-# Offense count: 94
+# Offense count: 95
 RSpec/MultipleExpectations:
   Max: 5
```

Comment:
Do you think we could ignore these? Couldn't quite figure out how to get them working nicely together, this was the furthest I got:
```
it 'produces a warning' do
        logger = instance_double(Logger)
        allow(logger).to receive(:warn)
        allow(client).to receive(:logger).and_return(logger)
        allow(client).to receive(:post)

        client.channels_archive(channel: 'test')
        expect(logger).to have_received(:warn).with(/
          ^channels\.archive:\ This\ method\ is\ deprecated
          .+
          Alternative\ methods:\ conversations\.archive\.
        /x)
      end
```
And then it gives me `Prefer receive for setting message expectations.` 😞

Reply:
I think ignoring expectations is fine. I usually just `-a` everything and selectively fix if I feel like it.